tests/intensive/sam_parity_tests.py

```
@@ -962,5 +962,310 @@ def test_interactive_box_point_combo_parity(self):
         )
 
 
+class TestSAM2InteractiveParity(unittest.TestCase):
+    """Compare predict_interactive outputs against direct
+    segment_anything inference using evaluate_detections."""
+
+    MODEL_NAME = "segment-anything-2-hiera-tiny-image-torch"
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.11)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)